features/projects/components/project-card/project-card.tsx

```diff
@@ -1,5 +1,5 @@
 import Link from "next/link";
-import capitalize from "lodash/capitalize";
+// import capitalize from "lodash/capitalize";
```

Comment:
I'd remove this line. Same in the test. I think you can also remove the `lodash/capitalize` package altogether from the dependencies